etc/env/configs/testnet.toml

```diff
@@ -0,0 +1,30 @@
+__imports__ = ["base", "l1-inits/testnet.env", "l2-inits/testnet.init.env" ]
+ZKSYNC_DEBUG_LOGS=true
+CHAIN_ETH_ZKSYNC_NETWORK_ID=300
+CONTRACTS_ERA_CHAIN_ID="300"
+ETH_CLIENT_WEB3_URL= ""
+# MISC_ETHERSCAN_API_KEY = ""
```

Comment:
above we use "" for fields that are to be set by the user explicitly, please do the same ehre